### ISSUE TITLE
Totally lost

### ISSUE THREAD

**Issue by scott-laj-pfe** [githubteacher/dev-dec-19#54]
@crichID 
COFFIS01@AMRLPF0088VW ~/Code/temp/github-games (gh-pages)
$ git branch --all
* gh-pages
  stats-update
  remotes/origin/HEAD -> origin/gh-pages
  remotes/origin/game-instructions
  remotes/origin/gh-pages
  remotes/origin/readme-update
  **remotes/origin/shape-colors**
  remotes/origin/stats-update
  remotes/upstream/game-instructions
  remotes/upstream/gh-pages
  **remotes/upstream/shape-colors**
  remotes/upstream/stats-update

**COFFIS01@AMRLPF0088VW ~/Code/temp/github-games (gh-pages)
**$ git checkout -b shape-colors origin/shape_colors**
fatal: Cannot update paths and switch to branch 'shape-colors' at the same time.
Did you intend to checkout 'origin/shape_colors' which can not be resolved as commit?**

**Comment by scott-laj-pfe** [comment]
Ok I found the cause of the fatal error.  It was a typo (shape_colors instead of shape-colors).  But I still don't understand what the purpose of that command was given that  origin and upstream already had shape-colors branches.  Choosing to create a local branch with the same name seems a bad practice if that was indeed what he was trying to accomplish.

**Comment by crichID** [comment]
> Ok I found the cause of the fatal error.

:sparkles: @scott-laj-pfe 

> I still don't understand what the purpose of that command was given that origin and upstream already had shape-colors branches.

The reason we use that longer command is because we needed to create a local copy of the shape-colors branch and we had to tell it to create the branch based on the one in origin/ instead of the one in upstream/.

> Choosing to create a local branch with the same name seems a bad practice if that was indeed what he was trying to accomplish.

You typically want your local branch to have the same name as the branch on the remote. What makes it confusing here is that there is more than one remote. Since you can't push to the upstream remote, then it only makes sense to make branches based on the origin remote.

**Comment by scott-laj-pfe** [comment]
@crichID Makes sense if you're a master at this stuff and it's EXPLAINED.  He's doing an AWFUL job today.  I've been using git (not github) for awhile and I still can't follow him today.

**Comment by crichID** [comment]
@scott-laj-pfe we appreciate your honest feedback! It is important in helping us continue to improve our delivery and our curriculum!
